halo/app/monitor.go

```diff
@@ -57,33 +64,38 @@ func monitorCometForever(
 }
 
 // monitorCometOnce monitors the cometBFT peers, and sync status.
-func monitorCometOnce(ctx context.Context, rpcClient rpcclient.Client, isSyncing func() bool, lastHeight int64) (int64, error) {
-	if netInfo, err := rpcClient.NetInfo(ctx); err != nil {
+func monitorCometOnce(ctx context.Context, rpcClient rpcclient.Client, isSyncing func() bool, lastHeight int64, status *readinessStatus) (int64, error) {
+	netInfo, err := rpcClient.NetInfo(ctx)
+	status.setConsensusP2PPeers(netInfo.NPeers)
+	if err != nil {
 		return 0, errors.Wrap(err, "net info")
 	} else if netInfo.NPeers == 0 {
 		log.Error(ctx, "Halo has 0 consensus p2p peers", nil)
 	}
 
-	setConstantGauge(cometSynced, !isSyncing())
+	synced := !isSyncing()
+	setConstantGauge(cometSynced, synced)
+	status.setConsensusSynced(synced)
 
 	abciInfo, err := rpcClient.ABCIInfo(ctx)
 	if err != nil {
 		return 0, errors.Wrap(err, "abci info")
 	} else if !isSyncing() && lastHeight > 0 && abciInfo.Response.LastBlockHeight <= lastHeight {
 		log.Warn(ctx, "Halo height is not increasing, evm syncing?", nil, "height", abciInfo.Response.LastBlockHeight)
+		status.setConsensusSynced(false)
```

Comment:
readiness ConsensusSynced should be aligned with `cometSynced` metric. Lets remove this for now rather.

Reply:
Done